kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/challenges/ProductException.java

```diff
@@ -0,0 +1,4 @@
+package com.kodilla.good.patterns.challenges;
+
+public class ProductException extends Exception {
```

Comment:
dobrze by było nadpisać metody klasy Exception, tak żeby message był customowy (np. Product does not exist).